CONTRIBUTING.md

```diff
@@ -104,17 +104,16 @@ For pull request acceptance, you should:
 3.  Add a note to `CHANGELOG.md` about the changes. Include a link to the
     pull request.
 
-## Tips
+## Tox Tips
 
-1.  The primary tox environments for `watchmaker` include:
+1.  The _primary_ tox environments for `watchmaker` include:
 
     *   `check`
     *   `docs`
     *   `py26`
     *   `py27`
-    *   `py33`
-    *   `py34`
     *   `py35`
+    *   `py36`
 
```

Comment:
In plain text, the spacing looks fine. Are you seeing the spacing difference in the rendered version? If so, that is unfortunately not something we can control (at least, not without mucking around in the css of the readthedocs template, which isn't really part of this project...)